test/controllers/api/v2/hostgroups_controller_test.rb

```diff
@@ -11,19 +11,42 @@ def models
     @hostgroup = hostgroups(:common)
   end
 
+  test_attributes :pid => '70fe5df8-8917-4a46-b37a-708f449fe749'
```

Comment:
This is new to me. Where's it defined? What's it do?

Reply:
Many thanks @akofink 

Please refer to this PR when it was first used in katello project https://github.com/Katello/katello/pull/7337#pullrequestreview-115674938

and to these PRs when it was introduced to foreman project 
https://github.com/theforeman/foreman/pull/5464
https://github.com/theforeman/foreman/pull/5586